src/internal.c

```diff
@@ -23397,7 +23404,7 @@ int CreateOcspResponse(WOLFSSL* ssl, OcspRequest** ocspRequest,
         createdRequest = 1;
         if (ret == 0) {
             ret = CreateOcspRequest(ssl, request, cert, der->buffer,
-                                                                   der->length);
+                      der->length, &ctxOwnsRequest);
```

Comment:
Can reuse createdRequest as freeRequest and thenCreateOcspRequest sets the value to 0 when ctx takes ownership.

Reply:
This saves a variable but makes the code slightly more cumbersome to read. Is it worthwhile?
(CreateOcspRequest should use inverted logic for the variable and can't set freeRequest to zero when  not taking ownership of the variable)